python/ray/_private/tracing_utils.py

```diff
@@ -0,0 +1,72 @@
+import inspect
+import os
+from typing import Callable, List
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter, SimpleSpanProcessor
+from ray._private.utils import import_attr
+from ray.serve.schema import TracingConfig
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
```

Comment:
btw, we need to make sure that `opentelemetry` is an optional dependency. if it is not installed we need to:

- Not raise any exceptions if the user doesn't use tracing config
- Raise a useful error telling them to install if they do use tracing config